src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -131,6 +166,54 @@ public void verifyTopLeaderboard() {
     verifyAdVisibleInSlot(presentLeaderboardSelector, presentLeaderboard);
   }
 
+  public void verifyFandomDesktopArticleTopLeaderboard() {
```

Comment:
Yep, we need to move these methods to the separate class or it will be a real mess.